app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java

```diff
@@ -42,11 +74,57 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View rootView = inflater.inflate(R.layout.fragment_climbs, container, false);
         ButterKnife.bind(this, rootView);
 
+        climbArrayList = new ArrayList<>();
+        mainUser = getArguments().getParcelable(HomePage.ARG_USER);
+        wallSectionIdNum = getArguments().getInt(WallSectionFragment.ARG_WALL_SECTION);
+        wallIdNum = getArguments().getInt(WallsFragment.ARG_WALL_ID);
+        getClimbsByWallSection(mainUser.getUserName(), wallSectionIdNum);
         recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
-        ClimbsRecyclerViewAdapter adapter = new ClimbsRecyclerViewAdapter(CLIMB_NUMS);
-        recyclerView.setAdapter(adapter);
-        recyclerView.setItemAnimator(new DefaultItemAnimator());
+
         return rootView;
     }
 
+    /**
+     * Method to make a network call to the Clamber Server to get all Climbs for the specified wall
+     * section. It also updates the adapter to pass through the returned list of climbs.
+     * @param userName - userName is used in the database query to return project and completed statuses
+     * @param wallSectionId - wallSectionId tells the query what climbs to return.
+     */
+    public void getClimbsByWallSection(String userName, int wallSectionId){
+        ConnectivityManager connectivityManager = (ConnectivityManager) getActivity().getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo networkInfo = connectivityManager.getActiveNetworkInfo();
+        if(networkInfo != null && networkInfo.isConnected()) {
+            Retrofit retrofit = new Retrofit.Builder()
+                    .baseUrl(HomePage.CONNECTION_WEB_ADDRESS)
+                    .addConverterFactory(JacksonConverterFactory.create())
+                    .build();
+            ClamberService clamber = retrofit.create(ClamberService.class);
+            final Call<List<Climb>> climbsCall = clamber.getClimbsByWallSection(userName, wallIdNum, wallSectionId);
+            climbsCall.enqueue(new Callback<List<Climb>>() {
+                @Override
+                public void onResponse(Call<List<Climb>> call, Response<List<Climb>> response) {
+                    if (response.code() == 200){
+                        List<Climb> climbs = response.body();
+                        for (int i = 0; i < climbs.size(); i++){
+                            climbArrayList.add(climbs.get(i));
+                        }
+
+                        ClimbsRecyclerViewAdapter adapter = new ClimbsRecyclerViewAdapter(climbArrayList);
+                        recyclerView.setAdapter(adapter);
+                        recyclerView.setItemAnimator(new DefaultItemAnimator());
+                    }
+
+
+                }
+
+                @Override
+                public void onFailure(Call<List<Climb>> call, Throwable t) {
+                    Log.d(TAG, "Failure: ", t);
```

Comment:
Give a little more context in the error message "Failure while attempting to get climbs by wall section" or something along those lines.